files/package.json

```diff
@@ -32,113 +32,107 @@
-        "ember-cli-fastboot": "2.2.3",
-        "ember-cli-htmlbars": "^5.3.2",
-        "ember-cli-inject-live-reload": "^2.0.2",
-        "ember-cli-mirage": "^2.2.0",
-        "ember-cli-moment-shim": "^3.7.1",
-        "ember-cli-notifications": "^6.2.0",
+        "ember-cli-fastboot": "3.3.0",
+        "ember-cli-htmlbars": "^6.0.1",
+        "ember-cli-inject-live-reload": "^2.1.0",
+        "ember-cli-mirage": "^2.4.0",
+        "ember-cli-notifications": "^8.0.0",
         "ember-cli-sass": "^10.0.1",
         "ember-cli-sri": "^2.1.1",
         "ember-cli-typescript": "^4.1.0",
         "ember-cli-typescript-blueprints": "^3.0.0",
         "ember-cli-uglify": "^3.0.0",
-        "ember-data": "~3.25.0",
-        "ember-data-storefront": "0.17.3",
-        "ember-error-tracker": "^1.2.0",
+        "ember-data": "~4.3.0",
+        "ember-data-storefront": "0.18.1",
+        "ember-error-tracker": "^1.2.5",
         "ember-export-application-global": "^2.0.1",
-        "ember-fetch": "^8.0.4",
-        "ember-intl": "^5.6.2",
+        "ember-fetch": "^8.1.1",
+        "ember-intl": "^5.7.2",
         "ember-load-initializers": "^2.1.2",
-        "ember-maybe-import-regenerator": "^0.1.6",
-        "ember-modifier": "^2.1.1",
-        "ember-moment": "^8.0.0",
-        "ember-page-title": "^6.2.1",
-        "ember-qunit": "^5.1.2",
-        "ember-resolver": "^8.0.2",
-        "ember-simple-auth": "^3.0.0",
-        "ember-source": "~3.25.4",
-        "ember-template-lint": "^2.18.1",
+        "ember-modifier": "^3.2.7",
+        "ember-page-title": "^7.0.0",
+        "ember-qunit": "^5.1.5",
+        "ember-resolver": "^8.0.3",
+        "ember-simple-auth": "^4.2.1",
+        "ember-source": "~4.3.0",
+        "ember-template-lint": "^4.3.0",
         "ember-truth-helpers": "^3.0.0",
-        "eslint": "^7.20.0",
-        "eslint-config-prettier": "^7.2.0",
-        "eslint-plugin-ember": "^10.2.0",
+        "eslint": "^7.32.0",
+        "eslint-config-prettier": "^8.5.0",
+        "eslint-plugin-ember": "^10.5.9",
         "eslint-plugin-node": "^11.1.0",
-        "eslint-plugin-prettier": "^3.3.1",
+        "eslint-plugin-prettier": "^4.0.0",
+        "eslint-plugin-qunit": "^7.2.0",
         "loader.js": "^4.7.0",
         "npm-run-all": "^4.1.5",
-        "prettier": "^2.3.0",
-        "qunit": "^2.14.0",
-        "qunit-dom": "^1.6.0",
-        "sass": "^1.32.8",
-        "true-myth": "^4.1.1",
-        "typescript": "^4.2.2"
+        "prettier": "^2.6.1",
+        "qunit": "^2.18.0",
+        "qunit-dom": "^2.0.0",
+        "sass": "^1.50.0",
+        "true-myth": "^5.2.0",
+        "typescript": "^4.6.3"
     },
     "engines": {
-        "node": "10.* || >= 12"
+        "node": "12.* || 14.* || >= 16"
     },
     "ember": {
         "edition": "octane"
     },
     "fastbootDependencies": [
-        "node-fetch",
-        "moment"
+        "node-fetch"
     ],
     "volta": {
         "node": "12.19.0"
```

Comment:
We probably should pin the node version to 14.* because the deployments are using 14 from now on. Also probably need to update the `.nvmrc` file too